core/templates/pages/topic-editor-page/navbar/topic-editor-navbar-breadcrumb.component.html

```diff
@@ -1,4 +1,4 @@
-<div class="nav navbar-nav oppia-navbar-breadcrumb">
+<div class="nav navbar-nav oppia-navbar-breadcrumb" *ngIf="topic!=null">
```

Comment:
`*ngIf="topic"` should suffice. Ditto elsewhere.

Reply:
Done